README.md

```diff
@@ -69,3 +69,80 @@ Example  `platform_name_prefix` for Dasharo supported platforms:
 * `Protectli_VP4630`
 * `Protectli_VP4670`
 * `Protectli_VP6670`
+
+## Dasharo Memory HCL generation
+
+The new feature allows the memory report to automatically update the hcl report
+tables. The changes are not committed though, leaving the option to review and
+fix the final result.
+The option is `-u` (`--update`)
+
+To generate memory hcl report:
+
+1. Do the same preparations as for CPU HCL (steps 1-3).
+
+2. Command:
+
+    ```bash
+    find . -name "Micro-Star_International_Co.,_Ltd.*.tar.gz" -print0 | xargs -0 -n1 bash -c './dts-hclmgr --quiet --force --update  memory "$0"'
+    ```
+
+3. Snippet of output:
+
+    ```bash
+    Modified: docs/docs/resources/hcl/memory/pro-z690-a-wifi-ddr4.md
+    Diff:
+    20a21
+    > | Kingston | KF3200C16D4/32GX | 32768 MB | 2400 MT/s (PC4-19200) | -/-/&#10004 | v0.4.0 | Dasharo HCL report |
+    End Diff
+    From #lines
+        41     768    4376
+    To #lines
+        42     788    4487
+    ```
+
+    Here:
+
+    1. Information about what file was modified:
+
+        `Modified: docs/docs/resources/hcl/memory/pro-z690-a-wifi-ddr4.md`
+
+    2. Diff from the changes:
+
+        ```bash
+        Diff:
+        20a21
+        > | Kingston | KF3200C16D4/32GX | 32768 MB | 2400 MT/s (PC4-19200) | -/-/&#10004 | v0.4.0 | Dasharo HCL report |
+        End Diff
+        ```
+
+    3. Information about how the number of line changed:
+
+        ```bash
+        From #lines
+            41     768    4376
+        To #lines
+            42     788    4487
+        ```
+
+4. After the script has finished, go to the `docs` directory:
+
+    ```bash
+    cd docs
+    ```
+
+5. Display the diff:
+
+    ```bash
+    git diff
+    ```
+
+6. Commit the desired file:
+
+    ```bash
+    git add <path-to-file>
+    ```
+
+    ```bash
+    git commit -sm "<COMMIT_MESSAGE>"
+    ```
```

Comment:
Ok, do we plan to tell what the recommended set of usage models is? I would like to avoid a situation in which @Stanislaw-bnk has to ask what parameters are correct. We have just one use case, so there should not be much choice for parameters.

Reply:
@EduKav1813 @Stanislaw-bnk did we adressed it anywher?